### ISSUE TITLE
Outdated README

### ISSUE THREAD

**Issue by wlnirvana** [gluonhq/ChatApp#33]
The prerequisites section does not apply to latest version any more. The app has been deprecated by web app.

**Comment by abhinayagarwal** [comment]
Hi @wlnirvana ,

Can you please explain what do you mean by "app has been deprecated by web app" ?

**Comment by wlnirvana** [comment]
Hmm... That is a very good question.

I think I was talking about the "Install and open the Dashboard" in the README.md file. As far as I can recall, the Dashboard used to be an app that needs a native installation, but now should be accessed in browser.

**Comment by abhinayagarwal** [comment]
Yes, you are right. The Cloudlink application has been super-seeded with a webapp available at https://gluon.io.

Can you create a PR to fix this?
